main.go

```diff
@@ -181,18 +194,31 @@ func (h *Hookbot) KeyChecker(wrapped http.Handler) http.HandlerFunc {
 }
 
 func (h *Hookbot) ServePublish(w http.ResponseWriter, r *http.Request) {
-	message, err := ioutil.ReadAll(r.Body)
+	body, err := ioutil.ReadAll(r.Body)
+
 	if err != nil {
 		log.Printf("Error serving %v: %v", r.URL, err)
 		return
 	}
-	h.message <- message
+
+	url := r.URL
+	// The suffix is everything after the "/notify/"
+	p := strings.SplitN(url.Path, "/", 3)
+	suffix := p[2]
+	log.Printf("suffix: [[[%v]]]", suffix)
```

Comment:
please remove the log printf and (optionally) add a more "proper-looking" log message instead. (I'd be okay with hookbot logging messages it got to its stderr).